src/plugins/news/screens/article/index.jsx

```diff
@@ -26,8 +26,8 @@ class ArticleScreen extends AbstractNodeScreen {
   };
 
   getFormRenderProps() {
-    const { blocksmithState } = this.props;
-    return { blocksmithState };
+    const { blocksmithState, getNodeRequestState } = this.props;
+    return { blocksmithState, getNodeRequestState };
```

Comment:
@rsumilang i think you need this in a lot of other places? yes - all the non article history tabs are crashing